purchase_request_procurement/test/purchase_request_from_procurement.yml

```diff
@@ -21,11 +21,11 @@
   I check request details which created after run procurement.
 -
   !python {model: procurement.order}: |
-    procurement_ids = self.search(cr, uid, [('request_id','!=', False)])
-    for procurement in self.browse(cr, uid, procurement_ids, context=context):
+    procurement_ids = self.search([('request_id','!=', False)])
+    for procurement in self.browse(procurement_ids):
         request = procurement.request_id
         assert len(request.line_ids) == 1, "Request Lines should be one."
         line = request.line_ids[0]
-        assert line.product_id.id == procurement.product_id.id, "Product is not correspond."
-        assert line.product_uom_id.id == procurement.product_uom.id, "UOM is not correspond."
-        assert line.product_qty == procurement.product_qty, "Quantity is not correspond."
+        assert line.product_id.id == procurement.product_id.id, "Product does not correspond."
```

Comment:
I think there is not goal in mantaining the yml tests in v10. This is a good moment to get rid of this IMO.